src/status_im/contexts/shell/qr_reader/sheets/scanned_wallet_address.cljs

```diff
@@ -1,33 +1,40 @@
 (ns status-im.contexts.shell.qr-reader.sheets.scanned-wallet-address
   (:require
     [quo.core :as quo]
-    [status-im.contexts.wallet.common.utils :as utils]
-    [status-im.contexts.wallet.common.utils.networks :as network-utils]
+    [react-native.clipboard :as clipboard]
     [status-im.feature-flags :as ff]
     [utils.i18n :as i18n]
     [utils.re-frame :as rf]))
 
+(defn- on-press
+  [address]
+  ;; Originally, the flow went to the send flow, but it has been removed to avoid bugs. Please check
+  ;; https://github.com/status-im/status-mobile/issues/20972. The previous code has been commented
+  ;; out.
+  (clipboard/set-string address)
+  (rf/dispatch [:toasts/upsert
+                {:type :positive
+                 :text (i18n/label :t/address-copied)}])
+  #_(let [[_ split-address] (network-utils/split-network-full-address address)]
```

Comment:
If we want to leave the commented piece in the code for future use, I'd suggest to make refactoring that will help next developer to understand what happens:
- rename `on-press` to more meaningful `copy-address`
- create `send-to-address` handler, move there old code and comment it out
- leave send section in action-drawer with `:on-press send-to-address` and comment it out

With this changes the future developer will quickly grasp what is going on here and will need only to uncomment things to get it back to work, wdyt?

Reply:
Thanks for the suggestion @vkjr